revelations/perspire-demo/valaa.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Hmmh. 
This file should be modified to support dynamic inspire.js generation as part of a test runner or similar so that inspire & inspire-plugin .js files can be removed from git (the latter of which doesn't belong in @valos/vault in the first place).

Reply:
Anyway, once you're finished with the other changes, we can pair program the proper perspire test/demo runner valma scripts.

Reply:
removed inspire-plugin.